src/components/expandableSection/index.tsx

```diff
@@ -73,5 +83,9 @@ export default ExpandableSection;
 const styles = StyleSheet.create({
   container: {
     overflow: 'hidden'
+  },
+  separator: {
+    ...Dividers.d10,
+    marginLeft: Spacings.s10
```

Comment:
You can move the margin to a modifier and pass the dividers object without spreading, I think it's better :) 
```suggestion
  separator: Dividers.d10
```